_includes/js/objects.md

```diff
@@ -249,6 +249,12 @@ var playerName = gameScore.get("playerName");
 var cheatMode = gameScore.get("cheatMode");
 ```
 
+Alternatively, the `attributes` property of the `Parse.Object` can be treated as a Javascript object, and even destructured. 
+
+```javascript
+var { score, playerName, cheatMode } = result.attributes;
```

Comment:
Could you please use `const` instead of `var`?

Reply:
Yeah, I thought about that, but the whole documentation is using `var` so I wanted to be consistent.

Reply:
Thanks. We will work to fix the existing documentation.